experimental/FTheoryTools/src/AbstractFTheoryModels/methods.jl

```diff
@@ -624,6 +630,40 @@ function set_zero_section(m::AbstractFTheoryModel, desired_value::Vector{String}
   set_attribute!(m, :zero_section => [f(eval_poly(l, R)) for l in desired_value])
 end
 
+function set_gauge_algebra(m::AbstractFTheoryModel, algebras::Vector{String})
+gauge_algebras = [];
```

Comment:
Please indent the content of your functions, and avoiding trailing semicolons.

Also, initializing this to `[]` results in a `Vector{Any}` -- but perhaps a stricter type can be used?